src/enclave_proc/mod.rs

```diff
@@ -309,6 +313,32 @@ fn handle_command(
                     .set_action("Describe Enclaves".to_string())
             })?;
 
+            // Evaluate thread result at first describe, then set thread to None.
+            if pcr_thread.is_some() {
+                enclave_manager
+                    .set_measurements(
+                        pcr_thread
+                            .take()
+                            .unwrap()
```

Comment:
please do not use unwrap.

Reply:
If this join takes a long time, is there a risk for describe enclaves to timeout ?

Reply:
I would like to see some measurements regarding what happens when running this with a larger EIF. E.g 4Gib one.
I'm interested in the time it is taking for this and the memory footprint.